tests/posts_tests/test_posts.py

```diff
@@ -1,5 +1,5 @@
 import pytest
-from posts.models import Post, Resume, Rating, Poll, PollFile
+from posts.models import Post, Resume, Rating, Poll, PollFile, Choice
 from django.contrib.auth.models import User
 from django.core.exceptions import ValidationError
 from conftest import new_poll, new_user
```

Comment:
Not related to this PR, \
I guess you import `conftest` so you can use `new_poll` and `new_user` in parametrize
It should be `from tests.conftest import new_poll, new_user`